pkg/config/aws/aws.go

```diff
@@ -33,23 +35,39 @@ const (
 	SecretAccessKey = "AWS_SECRET_ACCESS_KEY"
 	// SessionToken represents AWS Session Key
 	SessionToken = "AWS_SESSION_TOKEN"
+
+	assumeRoleDuration = 25 * time.Minute
 )
 
 // GetConfig returns a configuration to establish AWS connection, connected region name and the role to assume if it exists.
-func GetConfig(config map[string]string) (awsConfig *aws.Config, region string, role string, err error) {
+func GetConfig(ctx context.Context, config map[string]string) (awsConfig *aws.Config, region string, err error) {
```

Comment:
This doc might also need update. It doesn't return the role but assumes it if it exists in config.

Reply:
Updated the comment!